### ISSUE TITLE
Zoom / unzoom hotkeys stopped working.

### ISSUE THREAD

**Issue by Pragma77** [fontforge/fontforge#5323]
* Provide a general summary of the issue in the **Title** above.  
* Before you open an issue, please check if a similar issue already exists or has been closed before.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### When reporting a bug/issue:
- [ ] Screenshot
- [x] The FontForge version and the operating system you're using
- [x] The behavior you expect to see, and the actual behavior
- [ ] Steps to reproduce the behavior 
- [x] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [ ] A description of the problem you're trying to solve, including _why_ you think this is a problem
- [ ] If the feature changes current behavior, reasons why your solution is better
- [ ] \(optional) Possible solution/fix/workaround

Fontforge 20230101
O/S Win 7
Was editing a glyph in outline window and Z and X stopped working, Zoom and ALT zoom functions using mouse could be selected and were working. I was moving between outline windows so not sure when X and Z stopped, but it stopped during my editing session between these windows only.
Selected View >Zoom out and zoom in, both worked fine.
After selecting zoom via the View menu, Z and X worked fine.
I then noticed that Z and X had appeared in the metrics window that was opened, but only one instance of each.
It is as if focus for the mouse was in one window, and the focus for the keyboard was in another. I have no idea if this is even possible.

**Comment by tuffnatty** [comment]
> It is as if focus for the mouse was in one window, and the focus for the keyboard was in another.

Yes, I am also seeing this behaviour both with 20230101 and with the latest code, on KDE Plasma 6 / X11. After raising one of multiple FontForge windows, one has to click inside to make it grab the focus. 20230101 release works the same way, so it's a relatively old issue.

**Comment by JoesCat** [comment]
On November 14, 2025 10:55:03 AM Phil Krylov wrote:
> tuffnatty left a comment (fontforge/fontforge#5323)
> 
> > It is as if focus for the mouse was in one window, and the focus for
> > the keyboard was in another.
> 
> Yes, I am also seeing this behaviour. After raising one of multiple
> FontForge windows, one has to click inside to make it grab the focus.
> 20230101 release works the same way, so it's a relatively old issue.

It's normal Desktop behaviour (at least with KDE desktop).
You could have the mouse pointing in one window, and scroll items in that 
window, while typing in another. Clicking the window will focus the 
keyboard to that window.
The KDE Desktop have environment settings you can set.
Look for "System Settings -> Window behaviour.
I imagine other desktops have similar type settings that can be modified.

I think this is also default behaviour with Windows too, but it's been a 
while since I've used that, so I'm not 100% sure about this here.
